src/server/pachyderm_test.go

```diff
@@ -786,6 +786,97 @@ func TestPipelineJobDeletion(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestPutFileSplit(t *testing.T) {
```

Comment:
Shouldn't we also check that the children files are named the way they are supposed to?  Right now looks like we are only checking the number of children.